wagtail_localize/test/models.py

```diff
@@ -45,6 +51,23 @@ class TestSnippet(TranslatableMixin, ClusterableModel):
     ]
 
 
+@register_snippet
+class TestSnippetNoDraft(TranslatableMixin, ClusterableModel):
+    field = models.TextField(gettext_lazy("field"))
+    # To test field level validation of snippets
+    small_charfield = models.CharField(max_length=10, blank=True)
```

Comment:
I don't think we need this as the whole purpose for it is to test that we cannot save drafts.

I would go as far as making it a regular model (i.e. no `@register_snippet`). Snippets are supercharged Django models with an aim to replicate modeladmin in the Wagtail admin. 

Something like:

```python
class TestNoDraftModel(TranslatableMixin):
    field = models.CharField(max_length=10, blank=True)

    translatable_fields = [
        TranslatableField("field"),
    ]

    panels = [
        FieldPanel("field"),
    ]
```

should be plenty